### PR TITLE
feat: display the relative position of the grades

### PR DESCRIPTION
**Pull request by yudukikun5120**
This feature is opt-in (being turned off by default).

### CODE REVIEW

**Comment by yudukikun5120** [comment]
@mkobayashime Could I ask you for reconfiguring CodeClimate?

**Review by mkobayashime — APPROVED** [review]
LGTM!

**Comment by yudukikun5120** [comment]
Thank you for code reviewing! I’d merge this PR after CodeClimate code scanning because I’m apprehensive of violation.

**Comment by mkobayashime** [comment]
@yudukikun5120 
I've just realized Code Climate has stopped working as I transferred this repo to `manaba-enhanced-for-tsukuba` org. I configured the new repo in Code Climate, but you may need to push some new commits to trigger the CI run. My apologies for the inconvinience.

**Comment by yudukikun5120** [comment]
@mkobayashime Let me say a word of thanks!

File: src/contentScript/showRelativeGradesPosition.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { showRelativeGradesPosition } from "../methods/showRelativeGradesPosition"
+import { getStorage } from "../network/storage"
+
+getStorage({
```
[the remark this answers is not in the document]
Comment (yudukikun5120) [reply src/contentScript/showRelativeGradesPosition.ts:4]: @mkobayashime I think the manager for features something like `registerFeature("featureName")` may need.

**Pull request merged**